### PR TITLE
Annouce OpenStack.NET 1.4 release

### PR DESCRIPTION
**Pull request by carolynvs**
This post announces the 1.4 release of OpenStack.NET and its support for CDN. This also discusses our plans for the SDK in the next few months.

Please don't merge this until my other 2 PRs are merged as this references their content:
* https://github.com/rackerlabs/docs-developer-blog/pull/33
* https://github.com/rackerlabs/docs-quickstart/pull/9

## Additional Info
Author Bio: Carolyn Van Slyck is a software developer on the Rackspace Developer Experience team.
Social Media: My twitter handle is carolynvs
Redaction/Editing: Yes, please help make my post the best it can be!
Desired Publish Date: Monday July 13th 2015

### CODE REVIEW

File: _posts/2015-07-13-openstacknet-1.4-and-beyond.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+comments: true
+author: Carolyn Van Slyck
+published: true
+categories:
+    - OpenStack
+    - OpenStack.NET
+    - CDN
+---
+
+[OpenStack.NET 1.4.0](https://github.com/openstacknetsdk/openstack.net/releases/v1.4.0.0) has just been pushed out the door! This release has two things going for it: support for Content Delivery Networks (CDN) and it heralds some big changes for the future of OpenStack.NET.
```
Comment (catlook) [inline _posts/2015-07-13-openstacknet-1.4-and-beyond.md]: ... two things going for it: the release provides support for Content Delivery Networks (CDN), and it heralds some big ...

File: _posts/2015-07-13-openstacknet-1.4-and-beyond.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+<!-- more -->
+
+## CDN Support
+
+[Rackspace CDN](http://www.rackspace.com/cloud/cdn-content-delivery-network/features) allows you to add a CDN service to your existing website with a single API call. The service pulls content from your website and caches it on Akamai's global network. From there you can control caching rules, restrict access and purge cached content.
```
Comment (catlook) [inline _posts/2015-07-13-openstacknet-1.4-and-beyond.md]: .... restrict access, and purge cached ...

File: _posts/2015-07-13-openstacknet-1.4-and-beyond.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+Below is a small example of how to get started. The [QuickStart for Rackspace CDN](https://developer.rackspace.com/docs/cdn/getting-started/?lang=dot-net) has a complete walkthrough and you can download the example project from the [OpenStack.NET Demo repository](https://github.com/openstacknetsdk/Demos/tree/master/RackspaceQuickstart).
```
Comment (catlook) [inline _posts/2015-07-13-openstacknet-1.4-and-beyond.md]: The following small example helps you to get started. ...
...  walkthrough, and you can ...

File: _posts/2015-07-13-openstacknet-1.4-and-beyond.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+var serviceDefinition = new ServiceDefinition("Example CDN Service", "cdn",
+    domains: new[] {new ServiceDomain("www.example.com")},
+    origins: new[] {new ServiceOrigin("example.com")});
+string serviceId = await cdnService.CreateServiceAsync(serviceDefinition);
+string service = await cdnService.WaitForServiceDeployedAsync(serviceId);
+```
+
+## OpenStack.NET Today
+
+OpenStack.NET has been a labor of love, passed down from maintainer to maintainer, which is amazing. However this has given the SDK an inconsistent experience, as each developer did things a wee bit differently. Some services use async, some have a dozen optional parameters and some are missing useful methods such as revoking a token. This is not criticism, rather it is an acknowledgement that we have a ways to go.
```
Comment (catlook) [inline _posts/2015-07-13-openstacknet-1.4-and-beyond.md]: ... However, this has ...
... ptional parameters, and some are missing ...

File: _posts/2015-07-13-openstacknet-1.4-and-beyond.md [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+
+Here is what you can expect from new services going forward:
+
+ * **Async by Default** Synchronous extension methods will continue to be provided for those who need them.
+ * **Clean Method Signatures** It should be much easier to understand what is required and not deal with a dozen optional parameters.
+ * **Testable** The SDK now uses the standard HttpClient and with [Flurl it is very straightforward to fake API responses](http://tmenier.github.io/Flurl/testable-http/).
+
+My final goal is that the SDK should be welcome wherever .NET is: Windows Desktop, iOS, Android, Windows Phone, Mac and Linux. The SDK isn't quite there yet but we'll be taking advantage of [Microsoft's new direction for .NET](http://www.hanselman.com/blog/AnnouncingNET2015NETAsOpenSourceNETOnMacAndLinuxAndVisualStudioCommunity.aspx) which has been in development for a while and will be released at the end of the month. I'm looking forward to building demo applications which showcase OpenStack.NET on as many platforms as possible.
```
Comment (catlook) [inline _posts/2015-07-13-openstacknet-1.4-and-beyond.md]: ... Windows Phone, Mac, and Linux. ....
... there yet, but we'll be  ...
... Community.aspx), which has been in development ...

File: _posts/2015-07-13-openstacknet-1.4-and-beyond.md [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+
+The future for OpenStack.NET is very bright indeed and I hope you'll come along for the ride!
```
Comment (catlook) [inline _posts/2015-07-13-openstacknet-1.4-and-beyond.md]: ... bright indeed, and I hope you'll ....

[ ed. note: Thanks for a very cool post!  Feel free to use any of my edits you like and discard the rest! - Cat ]

Comment (carolynvs) [inline _posts/2015-07-13-openstacknet-1.4-and-beyond.md]: Thank you for the edits! I'll make the changes and update the PR. Obviously I need to brush up on my comma skills. :smile:

**Pull request merged**